src/Tribe/Metabox.php

```diff
@@ -474,7 +475,13 @@ public function has_permission( $post, $data, $nonce_action ) {
 			$post = get_post( $post );
 		}
 
-		return ! empty( $data['nonce'] ) && wp_verify_nonce( $data['nonce'], $nonce_action ) && current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
+		if ( empty( $data['nonce'] ) ) {
```

Comment:
I think you can do a small refactor here to make it a bit more compact

```suggestion
		if ( empty( $data['nonce'] ) || ! wp_verify_nonce( $data['action'], $nonce_action ) ) {
			return false;
		}
```